packages/lesswrong/components/form-components/UsersListEditor.jsx

```diff
@@ -81,20 +106,25 @@ class UsersListEditor extends Component {
   }
 
   render() {
+    const { classes } = this.props
+
     return (
-      <div className="users-list-editor">
-        {this.props.label && <h5>{this.props.label}</h5>}
-        <SortableList
-          axis="xy"
-          items={this.state.userIds}
-          onSortEnd={this.onSortEnd}
-          currentUser={this.props.currentUser}
-          removeItem={this.removeUserId}
-          shouldCancelStart={this.shouldCancelStart}
-        />
-        <Components.UsersSearchAutoComplete
-          clickAction={this.addUserId}
-        />
+      <div className={classes.root}>
+        {this.props.label && <label>{this.props.label}</label>}
```

Comment:
Deconstruct label at the top, so that you don't need to refer to this.props here